src/sentry/api/serializers/models/organization.py

```diff
@@ -1,11 +1,15 @@
+from typing import Any, Dict, FrozenSet, List, Mapping, MutableMapping, Sequence, cast
```

Comment:
If you import `from __future__ import annotations` then you can type things using the Python structures, so `dict[str, Any]` instead of `Dict[str, Any]`. It's not a deal breaker, but I'm trying to push people towards that.

Reply:
makes sense. addressed in [a234d8820492624fd3253565d7e1ec74c8067f](https://github.com/getsentry/sentry/pull/31787/commits/a234d8820492624fd3253565d7e1ec74c8067fef), let me know if that looks good.